### PR TITLE
Favourites list

### PR DESCRIPTION
**Pull request by mandeepsangha**

### CODE REVIEW

**Review by craig-dsilva — APPROVED** [review]

File: src/containers/Programs/Reducer.ts [hunk 1]
```diff
@@ -40,13 +41,15 @@ const programsInitialState = {
   information: [],
   searchedInformation: [],
   filteredInformation: [],
+  favouriteInformation: [],
```
Comment (RitaGlushkova) [inline src/containers/Programs/Reducer.ts]: I don't think we need this in reducers?

File: src/containers/Programs/index.tsx [hunk 2]
```diff
@@ -176,7 +178,7 @@ const Programs = ({
 
   useEffect(() => {
     if (information) setStateToRender(information);
-  }, [information]);
+  }, [information, stateToRender]);
```
Comment (RitaGlushkova) [inline src/containers/Programs/index.tsx:180]: Do we need the stateToRender in this useEffect as a dependency?

File: src/containers/Programs/index.tsx [hunk 3]
```diff
@@ -235,21 +237,42 @@ const Programs = ({
...
+                      stateToRender={stateToRender}
+                      setStateToRender={setStateToRender}
+                      onThumbnailSelection={handleUserSelection}
+                      isSelected={index === selectedProgramIndex}
+                      numberOfReviews={data.reviews}
+                      programId={data.id}
+                    />
+                  );
+                })}
+                {stateToRender.map((data, index) => {
```
Comment (RitaGlushkova) [inline src/containers/Programs/index.tsx]: Could you please look into this Thumbnail. I think it looks like a double return?

File: src/containers/Programs/index.tsx [hunk 4]
```diff
@@ -287,6 +310,7 @@ const mapStateToProps = (state: RootReducerInterface) => ({
   isLoadingPrograms: state.ProgramsReducer.programs.isLoadingPrograms,
   searchedInformation: state.ProgramsReducer.programs.searchedInformation,
   filteredInformation: state.ProgramsReducer.programs.filteredInformation,
+  favouriteInformation: state.ProgramsReducer.programs.filteredInformation,
```
Comment (RitaGlushkova) [inline src/containers/Programs/index.tsx]: I am not sure we use favourites via reducers.

**Review by RitaGlushkova — CHANGES_REQUESTED** [review]
@mandeepsangha I left some comments. Could you please have a look before we merge?

File: src/containers/Programs/Reducer.ts (hunk 1 above)
Comment (mandeepsangha) [reply src/containers/Programs/Reducer.ts]: i will remove

File: src/containers/Programs/index.tsx (hunk 2 above)
Comment (mandeepsangha) [reply src/containers/Programs/index.tsx:180]: yes as this updates the list

File: src/containers/Programs/index.tsx (hunk 3 above)
Comment (mandeepsangha) [reply src/containers/Programs/index.tsx]: yes, deleted

File: src/containers/Programs/index.tsx (hunk 4 above)
Comment (mandeepsangha) [reply src/containers/Programs/index.tsx]: will take this out

**Review by RitaGlushkova — APPROVED** [review]

**Pull request merged**